packages/desktop/components/popup/popups/BalanceBreakdownPopup.svelte

```diff
@@ -50,30 +50,34 @@
 
         let unavailableTotalAmount = $selectedAccount?.votingPower
         for (const [outputId, unlocked] of Object.entries(accountBalance?.potentiallyLockedOutputs ?? {})) {
-            if (!unlocked) {
-                const output = (await $selectedAccount.getOutput(outputId)).output
-
-                let type: string
-                let amount: bigint
-                if (output.type !== OutputType.Treasury) {
-                    const commonOutput = output as CommonOutput
-                    if (containsUnlockCondition(commonOutput.unlockConditions, UnlockConditionType.Expiration)) {
-                        type = UnavailableAmountType.Unclaimed
-                        amount = BigInt(output.amount)
-                    } else if (
-                        containsUnlockCondition(commonOutput.unlockConditions, UnlockConditionType.StorageDepositReturn)
-                    ) {
-                        type = UnavailableAmountType.StorageDepositReturn
-                        amount = getStorageDepositFromOutput(commonOutput)
-                    } else if (containsUnlockCondition(commonOutput.unlockConditions, UnlockConditionType.Timelock)) {
-                        type = UnavailableAmountType.Timelock
-                        amount = BigInt(output.amount)
-                    }
-                }
+            if (unlocked) {
+                continue
+            }
+
+            const output = (await $selectedAccount.getOutput(outputId)).output
+            if (output.type !== OutputType.Treasury) {
```

Comment:
It's not working with timelocks anymore. Change the line below:

```suggestion
            if (output.type === OutputType.Treasury) {
```